synfig-core/src/synfig/main.cpp

```diff
@@ -125,7 +128,23 @@ synfig::get_version()
 const char *
 synfig::get_build_date()
 {
-	return __DATE__;
+	const int max_date_length = 50;
+	static char date_str[max_date_length] = {0};
+
+	if (date_str[0] == 0) {
+		if (char* source_date_epoch = getenv("SOURCE_DATE_EPOCH")) {
+			std::istringstream iss(source_date_epoch);
+			std::time_t t;
+			iss >> t;
+			if (iss.fail()
+			    || !iss.eof()
+			    || !std::strftime(date_str, sizeof(date_str), "%x", std::localtime(&t))) {
+				    std::strncpy(date_str, _("Unknown date"), max_date_length-1);
```

Comment:
What about `"Unknown build date"`? 
I think it will be more clear.

Reply:
Done